tso_session/intro_TSO_data_pipeline.ipynb

```diff
@@ -37,13 +37,7 @@
     "6. [Final Steps: Tso3Pipeline()](#tso3pipe)<br>\n",
```

Comment:
<p>Here is the only cell that errors out. There may be a file missing from the Box folder that's downloaded earlier in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>